src/component/base/DraftEditor.react.js

```diff
@@ -154,21 +154,21 @@ class DraftEditor extends React.Component {
 
     this._onBeforeInput = this._buildHandler('onBeforeInput');
     this._onBlur = this._buildHandler('onBlur');
-    // this._onCharacterData = this._buildHandler('onCharacterData');
+    this._onCharacterData = this._buildHandler('onCharacterData');
     this._onCompositionEnd = this._buildHandler('onCompositionEnd');
     this._onCompositionStart = this._buildHandler('onCompositionStart');
     this._onCompositionUpdate = this._buildHandler('onCompositionUpdate');
-    // this._onCopy = this._buildHandler('onCopy');
-    // this._onCut = this._buildHandler('onCut');
-    // this._onDragEnd = this._buildHandler('onDragEnd');
-    // this._onDragOver = this._buildHandler('onDragOver');
+    this._onCopy = this._buildHandler('onCopy');
+    this._onCut = this._buildHandler('onCut');
+    this._onDragEnd = this._buildHandler('onDragEnd');
+    this._onDragOver = this._buildHandler('onDragOver');
     this._onDragStart = this._buildHandler('onDragStart');
     this._onDrop = this._buildHandler('onDrop');
-    // this._onInput = this._buildHandler('onInput');
+    this._onInput = this._buildHandler('onInput');
     this._onFocus = this._buildHandler('onFocus');
-    // this._onKeyDown = this._buildHandler('onKeyDown');
-    // this._onKeyPress = this._buildHandler('onKeyPress');
-    // this._onKeyUp = this._buildHandler('onKeyUp');
+    this._onKeyDown = this._buildHandler('onKeyDown');
+    this._onKeyPress = this._buildHandler('onKeyPress');
+    this._onKeyUp = this._buildHandler('onKeyUp');
```

Comment:
these are fine to put back in, we ignore them at a lower level (see the noop fn in android edit handler)